src/controllers/browser.ts

```diff
@@ -328,6 +328,7 @@ export async function initBrowser(
       return await puppeteer.connect({ browserWSEndpoint: options.browserWS });
     } else {
       console.log('aqui');
+      removeStoredSingletonLock(options.puppeteerOptions);
```

Comment:
This function of deleting the token should not be placed here because how would we be able to recover the token later? It's important to keep the token available for use. Perhaps it would be more appropriate to implement a deactivation or revocation function for the token if needed, instead of permanently deleting it.

Reply:
the function does not remove the existing tokens, it only removes the `SingletonLock` symbolic link that causes the lock on OSX and Linux platforms.

Reply:
@BLRocha  this is great ! thank you !! now I learn that I don't need to erase the 'tokens' folder, just the singletonLock, and I don't need to re-scan the QR ! 🤗🤗🤗

@jonalan7 I believe the following would be a good compromise:

in controllers/initializer.ts ,
```
    if (typeof browser === 'boolean') {
      spinnies.fail(`browser-${session}`, {
        text: `Error no open browser....`
      });
      removeStoredSingletonLock(options.puppeteerOptions);
      statusFind && statusFind('noOpenBrowser', session);
      return reject(`Error no open browser....`);
    }
```

Reply:
I'm thinking that it might be an issue with the new Headless mode they added to Chrome. You can find information about it at this link: https://developer.chrome.com/articles/new-headless/

I believe we should make "new" an option, not the default. I suggest using "true" or "false" for now, until we can investigate further